omod/src/main/webapp/resources/htmlforms/122a-HIVCare_ARTCard-SummaryPage.xml

```diff
@@ -44,10 +42,15 @@
             submitValStage=submitValStageART;
 
             submitValStage.push(dateValidator("90267", "encounterDate", "", message4, "", "greater_than", true));
-            submitValStage.push(dateValidator("90297", "encounterDate", "", message1, "", "less_than", true));
-            submitValStage.push(dateValidator("90299", "90297", "encounterDate", message2, message1, "less_than", false));
             submitValStage.push(dateValidator("99161", "90299", "encounterDate", message3, message1, "less_than", false));
-
+            if (submitValStage.indexOf(false) != -1) {
+                submitVal = false;
+                jq().toastmessage('showErrorToast', "Form Validation violated. Check for errors");
```

Comment:
@slubwama 

1. "Violated" I think It should be validated

2. Why return here when you still have other checks at the bottom?